src/librustdoc/html/layout.rs

```diff
@@ -98,7 +98,7 @@ crate fn render<T: Print, S: Print>(
                            placeholder=\"Click or press ‘S’ to search, ‘?’ for more options…\" \
                            type=\"search\">\
                 </div>\
-                <button class=\"help-button\">?</button>
+                <a href=\"javascript:;\" class=\"help-button\">?</a>
```

Comment:
Why not use `#`?

```suggestion
                <a href=\"#\" class=\"help-button\">?</a>
```

Reply:
Because it updates the URL, which we don't want.

Reply:
Ok, wasn't sure :)

Reply:
You did well asking, like that if someone else comes across this PR and wonders about the same thing, there is an answer! :smile: